requirements/dev.txt

```diff
@@ -18,3 +18,27 @@ django-sslserver==0.14
 
 # sha256: ku47hrXBkhwtnawhe--JkMX4Usi8TZAYyv4PPzuBzTo
 nosenicedots==0.5
+
+# sha256: jnbbT3Kv6QCHxHRntYjQ3j2_hvbj7nYqok7KqmyhuwM
+pre-commit==0.4.4
+
+# sha256: qRNwGDrqY8h9hIfns5ntLZmnwvFLEI0nwLyK2e9ZXZo
+aspy.yaml==0.2.1
+
+# sha256: 6gYn4M8e_0ZftBZrbt3sQW4dq48w1kucWz_6uJgHJmE
+cached-property==1.1.0
+
+# sha256: Epii8bL0xKe5IczNFZ5OQvbXsPt1yGwM3s_HHwYYM_o
+jsonschema==2.4.0
+
+# sha256: z-plZ1KzlnhB98MAocl1ZSYBPuGfOsR9srdxNP84tQk
+nodeenv==0.13.1
+
+# sha256: w2yTiocuX_SUk4szsUqqFWy0OexnVI_Ks1Nbt4sIRug
+pyyaml==3.11
+
+# sha256: PIjPffEUwyzwZR0S-Xm421VtmS1kdJPlvbvigovkAAc
+virtualenv==12.1.1
+
+# sha256: PkWmgeXnnrMfCUsxhDNG7Zo95Of2682hDHSLPEXFWX4
+simplejson==3.0.7
```

Comment:
I'm getting this error after installing everything in `requirements/dev.txt`:

```
 <fjord> (<detached:remotes/official/pr/547=105a3 fjord/ scripts/) saturn2 ~/mozilla/fjord> pre-commit install
Traceback (most recent call last):
  File "/home/willkg/.virtualenvs/fjord/bin/pre-commit", line 5, in <module>
    from pkg_resources import load_entry_point
  File "/home/willkg/.virtualenvs/fjord/lib/python2.7/site-packages/pkg_resources.py", line 2829, in <module>
    working_set = WorkingSet._build_master()
  File "/home/willkg/.virtualenvs/fjord/lib/python2.7/site-packages/pkg_resources.py", line 449, in _build_master
    ws.require(__requires__)
  File "/home/willkg/.virtualenvs/fjord/lib/python2.7/site-packages/pkg_resources.py", line 742, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/home/willkg/.virtualenvs/fjord/lib/python2.7/site-packages/pkg_resources.py", line 639, in resolve
    raise DistributionNotFound(req)
pkg_resources.DistributionNotFound: ordereddict
```

If I install `ordereddict`, then I hit this error:

```
<fjord> (<detached:remotes/official/pr/547=105a3 fjord/ scripts/) saturn2 ~/mozilla/fjord> pre-commit install
Traceback (most recent call last):
  File "/home/willkg/.virtualenvs/fjord/bin/pre-commit", line 5, in <module>
    from pkg_resources import load_entry_point
  File "/home/willkg/.virtualenvs/fjord/lib/python2.7/site-packages/pkg_resources.py", line 2829, in <module>
    working_set = WorkingSet._build_master()
  File "/home/willkg/.virtualenvs/fjord/lib/python2.7/site-packages/pkg_resources.py", line 449, in _build_master
    ws.require(__requires__)
  File "/home/willkg/.virtualenvs/fjord/lib/python2.7/site-packages/pkg_resources.py", line 742, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/home/willkg/.virtualenvs/fjord/lib/python2.7/site-packages/pkg_resources.py", line 639, in resolve
    raise DistributionNotFound(req)
pkg_resources.DistributionNotFound: argparse
```

If I install `argparse`, then things work.

So I think we need to add:

```
argparse==1.3.0
ordereddict==1.1
```

plus their SHA hashes.